### ISSUE TITLE
How to test the statement view

### ISSUE THREAD

**Issue by yifanhunter** [lixin4ever/BERT-E2E-ABSA#11]
How to test the statement view, for example, I input "this movie is very good", and feedback positive

**Comment by yifanhunter** [comment]
Excuse me again，what are the requirements for the tested data? I find that your data has been marked. Test data is also labeled.

**Comment by lixin4ever** [comment]
For your first issue, this project may not support the detection of the overall sentiment (or "view") of the input sentence.

For your second issue, the labels for testing data is not used under the evaluation/inference mode. You can set some dummy labels (e.g., randomly sample the labels from the valid tag set) for the input data.

**Comment by yifanhunter** [comment]
Thank you for your reply, and ask another question: Does your model do not support downstream tasks? That is, I give a standard text, and the corresponding emotional output cannot be achieved through the model. If it is not supported now, how to adjust the support in the future? Looking forward to your reply, thank you!

**Comment by yifanhunter** [comment]
Thank you for your reply, and ask another question: Does your model do not support downstream tasks? That is, I give a standard text, and the corresponding emotional output cannot be achieved through the model. If it is not supported now, how to adjust the support in the future? Looking forward to your reply, thank you!

**Comment by lixin4ever** [comment]
You mean give the input sentence and output the extracted aspect terms as well as the corresponding sentiments/emotions?

**Comment by yifanhunter** [comment]
Yes, give the system an unlabeled text, and get the aspects and emotions directly through the system.

**Comment by lixin4ever** [comment]
The current strategy in `work.py` is to read the input from the file (batch-based input). For adapting the code to the streaming input, you need to write a function to build the valid input, namely, "input_ids", "attention_mask" and "token_type_ids" (note that "labels" is not necessary and you can set this argument as None). If you have no idea on how to do the tokenization, please refer to the function [convert_examples_to_seq_features](https://github.com/lixin4ever/BERT-E2E-ABSA/blob/30b3dffa8676363afd210997efccaac3110ccecf/glue_utils.py#L191). 

Besides, you also need to add some post-processing function to convert the model output into the desired format.

**Comment by jiutongl** [comment]
Excuse me, has this problem been solved yet
